sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -0,0 +1,746 @@
+        >;
+
+        assert.isNotEmpty(pages);
+
+        for (const [fieldName, expectedValue] of Object.entries(expectedFieldValues) as [
+          keyof typeof expectedFieldValues,
+          string
+        ][]) {
+          const field = idDocument.fields[fieldName];
+          assert.equal(field?.value, expectedValue);
+        }
+
+        for (const [fieldName, expectedDate] of Object.entries(expectedDateValues) as [
+          keyof typeof expectedDateValues,
+          Date
+        ][]) {
+          const { value: date } = idDocument.fields[fieldName] as DocumentDateField;
+          assert.equal(date?.getDate(), expectedDate.getDate());
+          assert.equal(date?.getMonth(), expectedDate.getMonth());
+          assert.equal(date?.getFullYear(), expectedDate.getFullYear());
+        }
+      });
+
+      it("url", async () => {
+        const url = makeTestUrl("/license.jpg");
+
+        const poller = await client.beginAnalyzeDocuments(
+          PrebuiltModels.IdentityDocument,
+          url,
+          testPollingOptions
+        );
+        const {
+          documents,
+          documents: [idDocumentNaive],
+          pages,
+        } = await poller.pollUntilDone();
+
+        assert.isNotEmpty(documents);
+
+        assert.equal(idDocumentNaive.docType, "prebuilt:idDocument:driverLicense");
+
+        const idDocument = idDocumentNaive as Extract<
+          IdentityDocument,
+          { docType: "prebuilt:idDocument:driverLicense" }
+        >;
+
+        assert.isNotEmpty(pages);
+
+        for (const [fieldName, expectedValue] of Object.entries(expectedFieldValues) as [
+          keyof typeof expectedFieldValues,
+          string
+        ][]) {
+          const field = idDocument.fields[fieldName];
+          assert.equal(field?.value, expectedValue);
+        }
+
+        for (const [fieldName, expectedDate] of Object.entries(expectedDateValues) as [
+          keyof typeof expectedDateValues,
+          Date
+        ][]) {
+          const { value: date } = idDocument.fields[fieldName] as DocumentDateField;
+          assert.equal(date?.getDate(), expectedDate.getDate());
+          assert.equal(date?.getMonth(), expectedDate.getMonth());
+          assert.equal(date?.getFullYear(), expectedDate.getFullYear());
+        }
+      });
+
+      it("invalid locale throws", async () => {
+        const url = makeTestUrl("/license.jpg");
+
+        try {
+          const poller = await client.beginAnalyzeDocuments(PrebuiltModels.IdentityDocument, url, {
+            locale: "thisIsNotAValidLocaleString",
+            ...testPollingOptions,
+          });
+
+          await poller.pollUntilDone();
+          assert.fail("Expected an exception due to invalid locale.");
+        } catch (ex) {
+          // Just make sure we didn't get a bad error message
+          assert.isFalse((ex as Error).message.includes("<empty>"));
+        }
+      });
+    });
+  }).timeout(60000);
```

Comment:
is this on a `describe`? I thought it only work on `it`s.

Reply:
The types for mocha at least allow this, and it doesn't panic at runtime. I'm fairly certain it works by applying the timeout to the group of tests.